PostFreeSurfer/PostFreesurferPipelineLongPrep.sh

```diff
@@ -0,0 +1,434 @@
+
+#needed to transform Freesurfer mask to MNI space.
+opts_AddMandatory '--freesurferlabels' 'FreeSurferLabels' 'file' "location of FreeSurferAllLut.txt"
+opts_ParseArguments "$@"
+
+#display the parsed/default values
+opts_ShowValues
+
+# Naming Conventions
+T1wImage="T1w"
+T1wImageBrainMask="brainmask_fs"
+T1wFolder="T1w" #Location of T1w images
+T2wImage="T2w"
+T2wFolder="T2w" #Location of T2w images
+Modalities="T1w T2w"
+MNI_08mm_template="$HCPPIPEDIR/global/templates/MNI152_T1_0.8mm.nii.gz"
+
+IFS=',' read -r -a timepoints <<< "$Timepoints_cross"
+Timepoint_cross=${timepoints[0]}
+
+############################################################################################################
+# The next block computes the transform from T1w_acpc_dc (cross) to T1w_acpc_dc (long_template).
+Timepoint_long=$Timepoint_cross.long.$Template
+T1w_dir_cross=$StudyFolder/$Timepoint_cross/T1w
+T1w_dir_long=$StudyFolder/$Timepoint_long/T1w
+T1w_dir_template=$StudyFolder/$Subject.long.$Template/T1w
+LTA_norm_to_template=$T1w_dir_template/$Template/mri/transforms/${Timepoint_cross}_to_${Template}.lta
+
+if (( TemplateProcessing == 0 )); then #timepoint mode
+
+    log_Msg "Timepoint processing: computing the transform from T1w_acpc_dc (cross) to T1w_acpc_dc (longitudinal template)"
+    mkdir -p $T1w_dir_long/xfms
+    #Snorm=$T1w_dir_long/$Timepoint_long/mri/norm
+    Snorm=$T1w_dir_cross/$Timepoint_cross/mri/norm
+    mri_convert $Snorm.mgz $Snorm.nii.gz
+
+    T1w_cross=$T1w_dir_cross/T1w_acpc_dc_restore.nii.gz
+    T1w_long=$T1w_dir_long/T1w_acpc_dc_restore.nii.gz
+    T2w_long=$T1w_dir_long/T2w_acpc_dc_restore.nii.gz
+
+    #1. create resample transform from timepoints' T1w to 'norm' (equiv. to 'orig') space. This only applies reorient/resample, no actual registration
+    lta_convert --inlta identity.nofile --src $T1w_cross --trg $Snorm.nii.gz --outlta $T1w_dir_long/xfms/T1w_cross_to_norm.lta	
+
+    #2. concatnate previous transform with TP(orig)->template(orig) transform computed by FS.
+    mri_concatenate_lta $T1w_dir_long/xfms/T1w_cross_to_norm.lta $LTA_norm_to_template $T1w_dir_long/xfms/T1w_cross_to_template.lta
+    #3. Invert TP->norm transforms. Again, this is only reorient/resample, this isn't an actual registration transform.
+    lta_convert --inlta $T1w_dir_long/xfms/T1w_cross_to_norm.lta --outlta $T1w_dir_long/xfms/norm_to_T1w_cross.lta --invert
+    #4. Combine TP->template and orig->TP transforms to get TP->TP(HCP template) transform.
+    mri_concatenate_lta $T1w_dir_long/xfms/T1w_cross_to_template.lta $T1w_dir_long/xfms/norm_to_T1w_cross.lta $T1w_dir_long/xfms/T1w_cross_to_T1w_long.lta	
+    #5. convert the final TP_T1w->TP_T1w(HCP template) LTA transform to .mat(FSL).
+    lta_convert --inlta $T1w_dir_long/xfms/T1w_cross_to_T1w_long.lta --src $T1w_cross --trg $T1w_cross --outfsl $T1w_dir_long/xfms/T1w_cross_to_T1w_long.mat
+else #Template mode
+    :
```

Comment:
Why do you have an `else` section that does nothing?

Side note, `:` isn't obvious to me, I would use `true` if you really need a do-nothing command.

Reply:
Ok, removed this else.